### PR TITLE
Fix update actor failing because of a serializer error

### PR DESCRIPTION
**Pull request by BentiGorlich**
When there was a serializer error when updating the remote instance this error would cause the update actor to fail. We now surround it in a try/catch block so this doesn't happen anymore

### CODE REVIEW

File: src/Service/RemoteInstanceManager.php [hunk 1]
```diff
@@ -57,11 +57,18 @@ public function updateInstance(Instance $instance, bool $force = false): bool
...
+                /** @var NodeInfo $nodeInfo */
+                $nodeInfo = $serializer->deserialize($nodeInfoRaw, NodeInfo::class, 'json');
+                $instance->software = $nodeInfo?->software?->name;
+                $instance->version = $nodeInfo?->software?->version;
+            } catch (\Error|\Exception $e) {
+                $this->logger->warning('There as an exception decoding the nodeinfo from {url}: {e} - {m}', [
+                    'url' => $instance->domain,
+                    'e' => \get_class($e),
+                    'm' => $e->getMessage(),
+                ]);
```
Comment (melroy89) [inline src/Service/RemoteInstanceManager.php:70]: no need to throw up an error?

**Review by melroy89 — APPROVED** [review]

File: src/Service/RemoteInstanceManager.php (hunk 1 above)
Comment (BentiGorlich) [reply src/Service/RemoteInstanceManager.php:70]: nono if I throw that up then nothing has changed. The error can be catched here as this doesn't hinder the overall update process

Comment (melroy89) [reply src/Service/RemoteInstanceManager.php:70]: And actually catch the error higher in the chain..

Comment (BentiGorlich) [reply src/Service/RemoteInstanceManager.php:70]: I think this is the right place. The instance should be created, but the name and the version are just not essential...

**Pull request merged**